packages/table/src/Table.Interactions.stories.tsx

```diff
@@ -33,10 +33,6 @@ import {
 const meta: StoryMetaType<typeof Table> = {
   title: 'Components/Table/Interactions',
   component: Table,
-  // argTypes: {
-  //   shouldAlternateRowColor: { control: 'boolean' },
-  //   darkMode: { control: 'boolean' },
-  // },
```

Comment:
Should we just remove this?

Reply:
yeah, we can remove it.

Reply:
Ngl, it was early in the morning and I thought this was added in 😅. Ignore me lol